corehq/apps/smsbillables/async_handlers.py

```diff
@@ -138,8 +138,14 @@ def _directed_fee(direction, backend_api_id, backend_instance_id):
             return fmt_dollar_amount(usage_fee.amount + usd_gateway_fee)
 
         rate_table = []
+
+        from corehq.apps.sms.test_backend import TestSMSBackend
+
         for backend_instance in backends:
             backend_type = get_backend_by_class_name(backend_instance.doc_type)
+            # not a better way than hardcoding
+            if backend_type == TestSMSBackend:
```

Comment:
`backend_instance` is always of `SMSBackend` type. So `isinstance(backend_instance, SMSBackend)` won't work.